### PR TITLE
New package: gnuplot

### PR DESCRIPTION
**Pull request by peterbud**
This is a first version of the package using the mingw make file included in the package but this makefile has some limitations. Will look whether the default makefile can produce a better result

### CODE REVIEW

**Comment by peterbud** [comment]
Hmm, it was working in the CI [here](https://ci.appveyor.com/project/peterbud/mingw-packages/build/1.0.85)?

File: mingw-w64-gnuplot/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+prepare() {
+  cd "$srcdir"/${_realname}-${pkgver}
+  patch -p1 -i ${srcdir}/01-gnuplot.patch
+}
+
+build() {
+  cd "$srcdir"/${_realname}-${pkgver}
+  [[ -d "${srcdir}"/build-${CARCH} ]] && rm -rf "${srcdir}"/build-${CARCH}
+  #mkdir -p "${srcdir}"/build-${CARCH} && cd "${srcdir}"/build-${CARCH}
```
Comment (mati865) [inline mingw-w64-gnuplot/PKGBUILD]: Isn't is necessary?

File: mingw-w64-gnuplot/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Maintainer: Some One <some.one@some.email.com>
```
Comment (mati865) [inline mingw-w64-gnuplot/PKGBUILD]: Fix header.

File: mingw-w64-gnuplot/PKGBUILD (hunk 1 above)
Comment (peterbud) [reply mingw-w64-gnuplot/PKGBUILD]: I could not use mingw makefile in the package to build in a separate directory. Im working on the default makefile.am to make this work properly

**Pull request merged**